var/spack/repos/builtin/packages/py-types-pyaml/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyTypesPyaml(PythonPackage):
```

Comment:
I know it's still a draft, but note that the name of the package and the class should be `py-types-pyyaml` and `PyTypesPyyaml` instead of `py-types-pyaml` and `PyTypesPyaml`.

Reply:
Thanks Gianluca for spotting this. I'll fix it in the next commit.

Reply:
Done in the next commit.